keps/sig-network/563-dual-stack/README.md

```diff
@@ -1380,9 +1380,9 @@ This capability will move to beta when the following criteria have been met.
 This capability will move to stable when the following criteria have been met.
 
 * Support of at least one CNI plugin to provide multi-IP
-* e2e test successfully running on two platforms
+* e2e test successfully running on two platforms (Azure and kind)
 * testing ingress controller infrastructure with updated dual-stack services
```

Comment:
A specific test is not necessary because the dual-stack service object is backward-compat with existing ingress controllers.